src/pymovements/gaze/experiment.py

```diff
@@ -155,6 +231,7 @@ def pos2vel(
         ...     distance_cm=68,
         ...     origin='lower left',
         ...     sampling_rate=1000.0,
+        ...     eyetracker=None,
```

Comment:
Please leave out this line from the example, as this is already None by default.

But if you like, you can add another example to the docstring which doesn't pass any `sampling_rate` but an `EyeTracker` instance. In that case use simple one line descriptions like:

- initialize experiment by passing sampling_rate
- initialize experiment by passing an eye tracker instance